src/controllers/profile-settings.controller.js

```diff
@@ -0,0 +1,64 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+import JwtService from '../services/jwt.service';
+/**
+ *
+ *
+ * @class ProfileSettingsController
+ */
+class ProfileSettingsController {
+  /**
+  * users can create an account
+  * @static
+  * @description POST /api/view-profile
+  * @param {object} req request object
+  * @param {object} res response object
+  * @memberof ProfileSettingsController
+  * @returns {object} ResponseService
+  */
+  static async viewProfile(req, res) {
+    const userToken = JwtService.verifyToken(req.token);
+    const userRegistered = await UserService.findUserByProperty({ email: userToken.email });
+
+    delete userRegistered.dataValues.password;
+    delete userRegistered.dataValues.token;
+    delete userRegistered.dataValues.isVerified;
+    delete userRegistered.dataValues.createdAt;
+    delete userRegistered.dataValues.updatedAt;
+
+    ResponseService.setSuccess(200, 'User Profile Found', userRegistered);
+    ResponseService.send(res);
+  }
+
+
+  /**
+  * users can create an account
+  * @static
+  * @description POST /api/edit-profile
+  * @param {object} req request object
+  * @param {object} res response object
+  * @memberof ProfileSettingsController
+  * @returns {object} ResponseService
+  */
+  static async editProfile(req, res) {
+    const userToken = JwtService.verifyToken(req.token);
+    await Promise.all(Object.keys(req.body).map(async (key, index) => {
+      await UserService.updateUser(
+        { email: userToken.email },
+        { [key]: (Object.values(req.body)[index]).trim() }
+      );
+    }));
+    const userData = await UserService.findUserByProperty({ email: userToken.email });
+
+    delete userData.dataValues.password;
```

Comment:
can you use destructuring to get these values in fewer lines as possible? this is too many repetitions